### PR TITLE
BA-237 search enter focus state and add to pills too

### PR DESCRIPTION
**Pull request by Braedencraig**
add focus state to search enter and pills, ensure all search is tabbable

### CODE REVIEW

**Comment by lisafolkerson** [comment]
Heya! For this one can we toggle the pill selection on spacebar? Sorry it wasn't on the ticket but if we add it we don't have to come back and add later!

**Comment by Braedencraig** [comment]
@lisafolkerson I believe this one is ready for review again!

File: web/wp-content/themes/borealis-2021/src/js/scripts/checkbox-searchform.js [hunk 1]
```diff
@@ -58,9 +56,12 @@ export default function checkboxSearchForm(
     // Add event listener to checkboxes to updateUrl with term id's
     for (let i = 0; i < checkboxEls.length; i++) {
         checkboxEls[i].addEventListener("click", updateUrl);
+        checkboxElsLabels[i].addEventListener("keydown", updateUrl);
     }
 
     function updateUrl(e) {
+        // i
+        // console.log(e.keyCode === 32);
```
Comment (lisafolkerson) [inline web/wp-content/themes/borealis-2021/src/js/scripts/checkbox-searchform.js:64]: oh no! the functionality is all perfect but can you also remove these.

**Review by lisafolkerson — APPROVED** [review]
Approving! Just would like the commented-out console log removed please :)

**Pull request merged**